.gitignore

```diff
@@ -5,3 +5,5 @@
 .jekyll-metadata
 .sass-cache/
 _site/
+
+_config_localdev.yml
```

Comment:
It is OK to commit this file. If I recall correctly one can use it to build and serve locally.
(However, I'm pretty sure we can make due with a single one too.)

Reply:
> If I recall correctly one can use it to build and serve locally. (However, I'm pretty sure we can make due with a single one too.)

Yes, I created it for that purpose, to be able to correctly view locally without needing to execute `jekyll serve` (after much trial and error). I used that in turn to build the site such that I could push the files to a web location and have it work, as it did in the draft site. It boiled down to custom `base_url` and `url` values in `_config.yml`, and it's possible that they can be supplied as arguments to `jekyll build`.